ee/billing/quota_limiting.py

```diff
@@ -189,7 +194,7 @@ def org_quota_limited_until(
         return None
 
     # 1b. never drop or high trust
-    if organization.never_drop_data or trust_score == 15:
+    if (organization.never_drop_data and resource != QuotaResource.QAAS_READ_BYTES) or trust_score == 15:
```

Comment:
QaaS is about querying, not stoying.
Probably this should be some type of flag/property of a Resource. Not sure.

Reply:
Should we still keep this in because this is meant to not block high value customers, it's rarely used.

Reply:
Sure. I'm gonna revert, although it effectively makes the usage limits on this product ineffective for them.
My line of thinking was: usage limit on this endpoint will not cause any harm, they can go and rise it (also, by default it's high, isn't it?)